### PR TITLE
[WPF] Maps project for WPF backend

### PR DESCRIPTION
**Pull request by mohachouch**
### Description of Change ###

This PR adds support for Maps on WPF backend.

NOTE: This PR only includes the Xamarin.Forms.Maps.WPF project. Other PRs will include  nuspec, ControlGallery...

### API Changes (Xamarin.Forms.Core) ###

No changes in Xamarin Forms Core

### Behavioral Changes ###

You can create maps for WPF Backend

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Rebased on top of master at time of PR
- [ ] Changes adhere to coding standard
- [ ] Consolidate commits as makes sense

### CODE REVIEW

File: Xamarin.Forms.Maps.WPF/FormsPushPin.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+			Pin = pin;
+
+			UpdateLocation();
+			
+			Loaded += FormsPushPin_Loaded;
+			Unloaded += FormsPushPin_Unloaded;
+			MouseDown += FormsPushPin_MouseDown;
+		}
+
+		private void FormsPushPin_Loaded(object sender, System.Windows.RoutedEventArgs e)
```
Comment (rmarinho) [inline Xamarin.Forms.Maps.WPF/FormsPushPin.cs]: we don't use private

File: Xamarin.Forms.Maps.WPF/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
+﻿using System.Reflection;
+using System.Resources;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Windows;
+using Xamarin.Forms.Maps;
+using Xamarin.Forms.Maps.WPF;
+using Xamarin.Forms.Platform.WPF;
+
+// Les informations générales relatives à un assembly dépendent de
```
Comment (rmarinho) [inline Xamarin.Forms.Maps.WPF/Properties/AssemblyInfo.cs]: remove french comments

File: Xamarin.Forms.sln [hunk 3]
```diff
@@ -2572,6 +2574,58 @@ Global
...
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|x64.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|x64.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|x86.ActiveCfg = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.AppStore|x86.Build.0 = Release|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|ARM.ActiveCfg = Debug|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|ARM.Build.0 = Debug|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|iPhone.ActiveCfg = Debug|Any CPU
+		{89B0DB73-A32E-447C-9390-A2A59D89B2E4}.Debug|iPhoneSimulator.ActiveCfg = Debug|Any CPU
```
Comment (rmarinho) [inline Xamarin.Forms.sln]: Remove from iphone awn iphone simulator  build for release and debug

Comment (mohachouch) [reply Xamarin.Forms.sln]: I  think it's OK, I removed all lines with build.0 or Deploy.0 for debug and release.

Comment (rmarinho) [reply Xamarin.Forms.sln]: yeah it's working

**Pull request merged**